Packs/Netmiko/Integrations/Netmiko/README.md

```diff
@@ -1,120 +1,118 @@
+  This integration provides ssh-based access to network devices, servers, and other appliances that support this method of configuration. For a complete list of supported platforms, please visit the below URL:
 
   [Netmiko Platforms.md on Github](https://github.com/ktbyers/netmiko/blob/develop/PLATFORMS.md)
 
+## Configuration Parameters
+
+ - *Name* - Integration instance name
+ - *Platform* - The Netmiko-specific platform name
+ - *Hostname* - The IP address, Hostname, or FQDN to connect to over SSH
+ - *Port* - The port to use for the SSH connection
+ - *Credentials* - The credentials should be the same as the Tanium client.
+ - *Override the default timeout value* - Override the default read timeout value used for a SSH connection (useful for slow-responding devices). 
+
+**NOTE**: Platform names are taken from the supported
+[SSH](https://github.com/ktbyers/netmiko/blob/develop/PLATFORMS.md#supported-ssh-device_type-values) or [Telnet](https://github.com/ktbyers/netmiko/blob/develop/PLATFORMS.md#supported-telnet-device_type-values) device type lists on GitHub.
+
+
 ## Configure Netmiko Integration in Cortex XSOAR
 
- 1. Navigate to **Settings** > **Integrations**.
- 2. Search for **Netmiko**.
+ 1. Navigate to **Settings** - **Integrations**
+ 2. Search for **Netmiko**
  3. Click **Add instance** to create and configure a new integration instance.
 	 - *Name*: a name for the integration instance.
-	 - *Platform*: the platform identifier taken from the above SSH or Telnet platform name lists (e.g., linux_ssh, paloalto_panos, etc.).
+	 - *Platform*: the platform identifier taken from the above SSH or Telnet platform name lists (e.g., linux_ssh, paloalto_panos, etc.)
 	 - *Hostname*: The IP address, hostname, or FQDN for the device to connect to via SSH.
-	 - *Port*: The port to connect to via SSH.
-	 - *Credentials*: The username/password, or XSOAR credential object, to be used for the connection.
-	 **NOTE**: Platform names are taken from the supported [SSH](https://github.com/ktbyers/netmiko/blob/develop/PLATFORMS.md#supported-ssh-device_type-values) or [Telnet](https://github.com/ktbyers/netmiko/blob/develop/PLATFORMS.md#supported-telnet-device_type-values) device type lists on GitHub.
-
- 4. Click **Test** to validate the new instance. This performs a simple connection to the system hosting the SSH server.
+	 - *Port*: The port to connect to via SSH
+	 - *Credentials*: The username/password, or XSOAR credential object, to be used for the connection
+	 - *TimeoutOverride*: Override the timeout value (in seconds) for a given integration instance. This is useful for devices that are slow in responding with requested output over SSH.
+4. Click **Test** to validate the new instance. This performs a simple connection to the system hosting the SSH server.
```

Comment:
@ShirleyDenkberg is correct, yo do not need to have the parameters twice.
Please remove the section `## Configuration Parameters` and update `TimeoutOverride` to be `Override the default timeout value` to match the display name of the parameter.

Reply:
Fixed